stdlib/public/core/ImplicitlyUnwrappedOptional.swift

```diff
@@ -73,12 +73,11 @@ public enum ImplicitlyUnwrappedOptional<Wrapped>
   }
 
   /// Returns a mirror that reflects `self`.
-  public func _getMirror() -> _MirrorType {
-    // FIXME: This should probably use _OptionalMirror in both cases.
+  public func customMirror() -> Mirror {
```

Comment:
Doesn't the usual enum mirror produce the same output? I don't think Optional needs anything custom anymore.